exec/Chainweb/Env.hs

```diff
@@ -285,6 +286,8 @@ commands = hsubparser
        (progDesc "Serve the chainweb-data REST API (also does listen)"))
   <> command "fill-events" (info (FillEvents <$> bfArgsP <*> eventTypeP)
        (progDesc "Event Worker - Fills missing events"))
+  <> command "dedupe" (info (pure Dedupe)
```

Comment:
This was just to give me some flexibility to investigate the index stuff in more detail.

Reply:
Removed because the migration check made it impossible to complete the dedupe as a standalone operation and ultimately we only need it when `gaps` is run.